Pipfile

```diff
@@ -5,12 +5,13 @@ verify_ssl = true
 
 [dev-packages]
 moto = "*"
-boto3 = "*"
 pytest = "*"
 requests_mock = "*"
+boto3 = "*"
 
 [packages]
 requests = "*"
+jsonschema = "*"
 bleach = "*"
```

Comment:
Should set bleach to at least `3.1.4` to address current security advisories

Reply:
How would I do that?

Reply:
Take a read of https://realpython.com/pipenv-guide/ - A pythonista such as @bambrp can probably steer you better :)

Reply:
Alex I think you can do that with:
jsonschema = ">=3.1.4"

Reply:
Excuse my ignorance, but doesn't `*` mean "latest"? What's holding it back isn't the pipfile, but the pipfile.lock, which froze the numbers last time someone (probably me) did a `pipenv update`.
That's what needs to change- run the command to update pipfile.lock (you don't want to edit that one by hand), and commit it.

I think the benefits of the >= option is that if it can't find a compatible version that's high enough, it'd become a warning, so that might be worth doing too. But you have to do the update either way.

Reply:
@glanvl Do you mean bleach = ">=3.1.4"?

Reply:
Run pipenv update: didn't change the version to >=3.1.4, but run it with the "latest" setup

Reply:
Yeah I meant bleach sorry I was looking at the wrong bit when I put the example in. As per Phil's suggestion do we still want to change it to >=3.1.4 in the pipfile perhaps? It is set to 3.1.5 in the piplock file now at least.